resources/less/vars.less

```diff
@@ -28,6 +28,7 @@
 @input-font: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
 @modal-shadow: 0 1px 3px 0 rgba(0,0,0,0.30);
 @separator-color: #E6E6E6;
-
 @sg-color-red: #e44e4e;
 @sg-color-gray: #aeaeae;
+@disabled-gray: #9b9b9b;
```

Comment:
Not in the style guide. Although I might be able to get rid of these colors altogether, let's see...